### ISSUE TITLE
Is the api offline?

### ISSUE THREAD

**Issue by SomeoneCares11** [theapache64/GPix#6]
api currently offline

**Comment by fixy** [comment]
does not work for me too

**Comment by Dramex** [comment]
yeah, the project died months ago then the creator deiced to update it. in the last update its works for 1 month or so, then suddenly the project died again at Aug 20.
rip

**Comment by huyhai** [comment]
not work for me
